common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerOperationExecutor.java

```diff
@@ -126,7 +137,21 @@ public BrokerOperationExecutor(final @NonNull List<IIpcStrategy> strategies) {
         final List<BrokerCommunicationException> communicationExceptionStack = new ArrayList<>();
         for (final IIpcStrategy strategy : mStrategies) {
             try {
+                if (codeMarkerIsEnabled) {
```

Comment:
Hmmm I don't think having to brace each of these in a bool check is necessarily preferable - could we put a log function on/in the `CodeMarkerManager` itself? /cc @shahzaibj since he had similar feedback

Just thinking out loud:
- Potentially call the logger from the `CodeMarkerManager` itself, when markers are hit
- Conditionally log only if code markers are enabled
- `warn()` seems a bits severe; can it be `info()` or `verbose()`?